composition/file_content_loader.go

```diff
@@ -0,0 +1,51 @@
+package composition
+
+import (
+	"errors"
+	"fmt"
+	"github.com/tarent/lib-compose/logging"
+	"os"
+	"strings"
+	"time"
+)
+
+var RequestProcessorsNotApplicable = errors.New("request processors are not apliable on file content")
+
+type FileContentLoader struct {
+	parser ContentParser
+}
+
+func NewFileContentLoader() *FileContentLoader {
+	return &FileContentLoader{
+		parser: &HtmlContentParser{},
+	}
+}
+
+func (loader *FileContentLoader) Load(fd *FetchDefinition) (Content, error) {
+	if fd.RespProc != nil {
```

Comment:
I had the need for a different usecase, but I could also imagine that for error pages, which should be there always.